mission8/tests/auth.test.ts

```diff
@@ -0,0 +1,26 @@
+import request from 'supertest';
+
+import app from '../src/app';
+import prisma from '../src/utils/prisma';
+
+beforeAll(async () => {
+  await prisma.user.deleteMany();
+  await prisma.product.deleteMany();
+});
```

Comment:
통합 테스트는 격리되어 실행되어야 합니다. (DB등의 애플리케이션 외부 인프라를 사용하기 때문에)
afterAll에서도 전체 데이터를 삭제해주세요.

```suggestion
beforeAll(async () => {
  await prisma.user.deleteMany();
  await prisma.product.deleteMany();
});

afterAll(async () => {
  await prisma.user.deleteMany();
  await prisma.$disconnect();
});
```